code/datums/status_effects/debuffs/drunk.dm

```diff
@@ -188,8 +188,11 @@
 				to_chat(owner, span_warning("You're so tired... but you can't miss that shuttle..."))
 
 			else
-				to_chat(owner, span_warning("Just a quick nap..."))
-				owner.Sleeping(90 SECONDS)
+				blackout()
+
+/datum/status_effect/inebriated/drunk/proc/blackout()
+	var/mob/living/carbon/drunkyard = owner
+	drunkyard.gain_trauma(/datum/brain_trauma/severe/split_personality/blackout)
```

Comment:
This should be `TRAUMA_RESILIENCE_ABSOLUTE` so it's not fixed by surgery? 

(There should be an alternate way of waking up someone from a blackout, like splashing water on their face or something)